app/controllers/api/v1/short_form_controller.rb

```diff
@@ -11,8 +11,10 @@ def submit
     if short_form_validator.valid?
       application = custom_api_application_service.submit(application_api_params)
       logger.debug "application submit response: #{application}"
+      # if submitting a supplemental application we will re-fetch updated data from salesforce
       if params[:supplemental]
         application = soql_application_service.application(application[:id] || application_api_params[:id])
+        application['rental_assistances'] = soql_rental_assistance_service.application_rental_assistances(application[:id] || application_api_params[:id])
```

Comment:
is the lease pulled in in the application above?

Reply:
or it looks like it comes from a separate request?

Reply:
looks like in the supplementals controller we add the rental assistance into the application object the same way you're adding it here, so that's OK for now. Ideally I'd like to keep the application object as just the application object and treat rental assistances and leases separately, or at least group rental assistances and leases together since rental assistances are actually tied to a lease

Reply:
actually, can we just go ahead and at least move the rental assistances piece into the applications service for now so it's treated the same way as leases? pulling it out of here and the supplementals controller and putting it in the soql applications controller instead? would that break anything?